wp-content/themes/orient-theme/taxonomy-series.php

```diff
@@ -1,18 +1,35 @@
 <?php
 
-get_header(); ?>
+get_header();
 
+//create a copy of wp_query
+$copy = clone $wp_query;
 
+//alternative???
+// $copy = WP_Query($wp_query);  
+// $copy = new WP_Query( array('post_type' => 'post') );
+$copy->set('nopaging', true);
+$wp_query->set( 'nopaging' , true );
```

Comment:
Do we mean to set `nopaging` to `true` in both `$copy` and `$wp_query`? Or do we just want to set it to true for one of them?